modules/@apostrophecms/schema/index.js

```diff
@@ -1332,19 +1458,23 @@ module.exports = {
       // reasonable values for certain properties, such as the `idsStorage` property
       // of a `relationship` field, or the `label` property of anything.
 
-      validate(schema, options) {
+      validate(schema, options, parent = null) {
         schema.forEach(field => {
           // Infinite recursion prevention
           const key = `${options.type}:${options.subtype}.${field.name}`;
           if (!self.validatedSchemas[key]) {
             self.validatedSchemas[key] = true;
-            self.validateField(field, options);
+            self.validateField(field, options, parent);
```

Comment:
`relationship` field uses `validate`u an not `validateField` like does `array` and `object` fields.
Maybe the three should use the validate method since we can now pass the parent?